pkg/label/label.go

```diff
@@ -7,21 +7,24 @@
 package label
 
 const (
-	Signature = "containerd.io/snapshot/nydus-signature"
+	// Labels defined by CRI
+	CRIImageRef            = "containerd.io/snapshot/cri.image-ref"
+	CRIImageLayers         = "containerd.io/snapshot/cri.image-layers"
+	CRILayerDigest         = "containerd.io/snapshot/cri.layer-digest"
+	CRIManifestDigestLabel = "containerd.io/snapshot/cri.manifest-digest"
 
-	CRIImageRef    = "containerd.io/snapshot/cri.image-ref"
-	CRIImageLayers = "containerd.io/snapshot/cri.image-layers"
-	CRILayerDigest = "containerd.io/snapshot/cri.layer-digest"
-
-	ImagePullSecret   = "containerd.io/snapshot/pullsecret"
-	ImagePullUsername = "containerd.io/snapshot/pullusername"
-
-	TargetSnapshotLabel       = "containerd.io/snapshot.ref"
-	TargetManifestDigestLabel = "containerd.io/snapshot/cri.manifest-digest"
-	StargzLayer               = "containerd.io/snapshot/stargz"
+	// Labels defined by containerd
+	ImagePullSecret     = "containerd.io/snapshot/pullsecret"
+	ImagePullUsername   = "containerd.io/snapshot/pullusername"
```

Comment:
`ImagePullSecret` and `ImagePullUsername` are defined by nydus. :)

Reply:
So add Nydus prefix？

Reply:
Looks good to me.